pkg/controller/apischeme/apischeme_controller.go

```diff
@@ -61,6 +62,15 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	// Watch the Service we set up for rh-api
+	err = c.Watch(&source.Kind{Type: &corev1.Service{}}, &handler.EnqueueRequestForOwner{
+		IsController: true,
+		OwnerType:    &cloudingressv1alpha1.APIScheme{},
+	})
+	if err != nil {
+		return err
+	}
+
```

Comment:
Presumably because there are no owner references on that `Service`, this can go away.